package.json

```diff
@@ -66,6 +66,7 @@
     "react-datepicker": "^0.46.0",
     "react-dom": "^16.8.0",
     "react-dropzone": "^8.0.3",
+    "react-mentions": "^3.1.1",
```

Comment:
We added this but didn't update yarn.lock.

Reply:
Oh, sure.